docs/usage.rst

```diff
@@ -334,7 +334,14 @@ Examples
     -rw-r--r-- root   root       1383 May 22 22:25 etc/ImageMagick-6/colors.xml
     ...
 
-
+    $ borg list /mnt/backup::root-2013-08-02 --list-format="{mode} {user:6} {group:6} {size:8d} {isomtime} {path}{extra}{NEWLINE}"
+    drwxr-xr-x root   root          0 Jun 05 12:06 .
```

Comment:
May 31 20:40 is not iso format. but ok, that likely not only wrong in the stuff you added, but also in other examples that were not yet updated.